iina/uk.lproj/Localizable.strings

```diff
@@ -114,7 +114,7 @@
 "menu.volume_up" = "Гучність +%.0f% %";
 "osc_toolbar.full_screen" = "Повноекранний режим";
 "osc_toolbar.music_mode" = "Режим музики";
-"osc_toolbar.pip" = "Режим «Кадр у кадрі»";
+"osc_toolbar.pip" = "Режим «Картинка в картинці»";
```

Comment:
Wrong. The red one is used in official Ukrainian localisation of macOS apps (eg., Safari web browser).
<img width="292" alt="Screenshot 2019-05-06 16 38 04" src="https://user-images.githubusercontent.com/36358174/57229233-3ca09f00-701e-11e9-9104-12819044c9e9.png">